base/abstractset.jl

```diff
@@ -245,6 +245,7 @@ function ⊇ end
     ⊇(b, a) -> Bool
 
 Determine whether every element of `a` is also in `b`, using [`in`](@ref).
+Here are some operators related to subset: [`⊊`](@ref), [`⊈`](@ref).
```

Comment:
I think the phrasing in many other docstrings is this, possibly with more explanation inserted:
```suggestion

See also [`⊊`](@ref), [`⊈`](@ref).
```